build on Windows

Hi, 

I am trying to build this project on Windows 11 (Visual Studio 2022).

I had an error like this:
```
CMake Error at C:/autoware/install/share/pcl_ros/cmake/export_pcl_rosExport.cmake:104 (message):
  The imported target "pcl_ros::pcl_ros_filters" references the file

     "C:/autoware/install/lib/pcl_ros_filters.lib"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "C:/autoware/install/share/pcl_ros/cmake/export_pcl_rosExport.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  C:/autoware/install/share/pcl_ros/cmake/ament_cmake_export_targets-extras.cmake:9 (include)
  C:/autoware/install/share/pcl_ros/cmake/pcl_rosConfig.cmake:41 (include)
  CMakeLists.txt:15 (find_package)

```
I found that I need to change the lib type from `SHARED` to `STATIC`:

```cmake
add_library(pcl_ros_filters STATIC

add_library(pcd_to_pointcloud_lib STATIC tools/pcd_to_pointcloud.cpp)
```

--- a/pcl_ros/CMakeLists.txt
+++ b/pcl_ros/CMakeLists.txt
@@ -7,6 +7,7 @@ endif()
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic -fPIC)
 endif()
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 
 ## Find system dependencies
 find_package(Eigen3 REQUIRED)

